samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -0,0 +1,493 @@
+   * Data-Only connection: true
+   * No Personalized Name: false
+   * Find My Device: true
+
+For details, see :ref:`ug_bt_fast_pair_provisioning` in the Fast Pair user guide.
+
+.. include:: /applications/nrf_desktop/description.rst
+   :start-after: nrf_desktop_fastpair_important_start
+   :end-before: nrf_desktop_fastpair_important_end
+
+.. tip::
+   The sample encodes TX power in the advertising set data that includes the Fast Pair payload.
+   There is no need to provide the TX power value during device model registration.
+
+User interface
+**************
+
+The user interface of the sample depends on the hardware platform you are using.
+
+Development kits
+================
+
+LED 1:
+   Keeps blinking at constant intervals to indicate that firmware is running.
+
+LED 2:
+   Indicates that the ringing action is in progress.
+   Depending on the ringing state:
+
+   * Lit if the device is ringing.
+   * Off if the device is not ringing.
+
+LED 3:
+   Depending on the FMDN provisioning state:
+
+   * Lit if the device is provisioned.
+   * Off if the device is not provisioned.
+
+LED 4:
+   Depending on the states of the recovery mode and the identification mode:
+
+   * Lit if both modes are active on the device.
+   * Off if both modes are inactive on the device.
+   * Blinks at a 0.5 second interval if the identification mode is active on the device and the recovery mode is inactive on the device.
+   * Blinks at a 1 second interval if the recovery mode is active on the device and the identification mode is inactive on the device.
+
+Button 1:
+   Toggles between different modes of the Fast Pair advertising set:
+
+   * Without an Account Key stored on the device:
+
+     * Fast Pair advertising disabled.
+     * Fast Pair discoverable advertising (the default after the system bootup).
+
+   * With an Account Key stored on the device:
+
+     * Fast Pair advertising disabled.
+     * Fast Pair not discoverable advertising (the default after the system bootup).
+
+   .. note::
+      The Bluetooth advertising is active only until the Fast Pair Provider connects to a Bluetooth Central.
+      After the connection, you can still switch the advertising modes, but the switch will come into effect only after disconnection.
+
+      The sample automatically switches to the following Fast Pair advertising modes under certain conditions:
+
+      * Fast Pair not discoverable advertising - On the Account Key write operation.
+      * Fast Pair advertising disabled:
+
+        * Right before the factory reset operation.
+          Press **Button 1** to set the device to the Fast Pair discoverable advertising mode after the unprovisioning operation.
+        * After the beacon clock synchronization.
+
+Button 2:
+   Stops the ongoing ringing action.
+
+Button 3:
+   Decrements the battery level by 10% (the default value in the :kconfig:option:`CONFIG_APP_BATTERY_LEVEL_DECREMENT` Kconfig option), starting from the full battery level of 100%.
```

Comment:
The Kconfig don't link. Could you check if its correct?

Reply:
It is defined in this file:

https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bluetooth/fast_pair/locator_tag/src/battery/Kconfig.battery#L12

It seems that Kconfigs defined at the sample level are not visible and this is a general limitation of our documentation build system.

Reply:
Actually, the sample specific configuration are defined in the sample documentation. and provide syntax as :ref:`CONFIG_APP_BATTERY_LEVEL_DECREMENT<CONFIG_APP_BATTERY_LEVEL_DECREMENT>` . For example , https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/dect/dect_phy/hello_dect/README.html#configuration-options.

Reply:
Thank you, I wasn't aware of it. I need to add a dedicated configuration section to the sample.